src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -144,4 +145,25 @@ public String getOgcCollectionFeature(
 		return rtn;
 	}
 
+	@Operation(description = "Return data sets available at the monitoring location.", responses = {
+			@ApiResponse(responseCode = "200", description = "available datasets.", content = @Content(schema = @Schema(implementation = ObservationsJSON.class))),
+			@ApiResponse(responseCode = "404", description = "The specified collection monitoring location was not found.", content = @Content()) })
+	@GetMapping(value = "collections/{collectionId}/items/{monitoring-location-identifier}/observations", produces = MediaType.APPLICATION_JSON_VALUE)
+	public String getObservationTypes(
+			@RequestParam(value = "f", required = false, defaultValue = "json") String mimeType,
+			@PathVariable(value = "collectionId") String collectionId,
+			@PathVariable(value = "monitoring-location-identifier") String monLocIdentifer,
+			HttpServletResponse response) {
+
+		String rtn = collectionsDao
```

Comment:
Two issues:

1. Overloading of rtn, a large string gets created to validate the collectionId/featureId and then another string is created to contain the actual output we want.
2. A potentially database intensive query is run to validate the collectionId/featureId. It will probably be better in the long term to add the 'validation' logic to the actual query getting the output.

Reply:
Since both requests use the same params, just the second call could be made. If the second call is "expensive" then the first call should be a new call to exists in the dao.